src/org/labkey/test/components/ui/grids/EditableGrid.java

```diff
@@ -574,15 +575,22 @@ public List<String> getFilteredDropdownListForCell(int row, String columnName, @
         return lookupSelect.getOptions();
     }
 
+
+    public EditableGrid pasteFromCell(int row, String columnName, String pasteText)
+    {
+        return pasteFromCell(row, columnName, pasteText, true);
+    }
+
```

Comment:
Because this is a widely used method I would suggest that the validate parameter default to false.
Why?
This is the current signature, which means that current test using this method are not expecting a validation check. This changes the default behavior.
Defaulting to true could cause problems for Date, Time or DateTime fields. The string passed in may not match the format of the fields but it would still be a valid string to paste.

If we are going to make a global change like this can we _**please finally**_ change the name of this method? The action is not pasting from a cell it is pasting into a cell.

Reply:
I'm fine with this as opt-in behavior. I pushed the change this way to get results with this validation, since we already know what results will look like without validating.
I'm cool with renaming the method (LOL this will make the change very large) but if we want that, cool.